### PR TITLE
Slightly update README

### PR DESCRIPTION
**Pull request by vixentael**
I present you the gorgeous creation by @ilammy:

[Closed pull requests with Bitcode-related changes](https://github.com/cossacklabs/themis/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Bitcode+%F0%9F%90%99%22)

## Checklist

- [x] @ilammy likes this change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,14 +93,14 @@ Themis is available for the following languages/platforms, refer to [language ho
...
-Themis supports following CPU architectures: x86_64/i386, ARM, various Android architectures.
+Themis supports following CPU architectures: x86_64/i386, ARM, Apple Silicon (ARM64), various Android architectures.
 
 We build and verify Themis on the latest stable OS versions:
 
   - Debian (9, 10), CentOS (7, 8), Ubuntu (16.04, 18.04, 20.04)
   - macOS (10.12–10.15, 11)
   - Android (4–11)
-  - iOS and iPadOS (9–14beta),
+  - iOS (10–14) <img src="https://img.shields.io/github/issues-pr-closed/cossacklabs/themis/Bitcode%20%F0%9F%90%99?color=%23cceaff&label=Bitcode-related%20issues" alt="Bitrise"></img>
```
Comment (ilammy) [inline README.md:103]: Badges are typically links, so I'd suggest
```suggestion
  - iOS (10–14) [![Closed pull requests with Bitcode-related changes](https://img.shields.io/github/issues-pr-closed/cossacklabs/themis/Bitcode%20%F0%9F%90%99?color=informational&label=Bitcode-related%20issues)](https://github.com/cossacklabs/themis/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Bitcode+%F0%9F%90%99%22)
```

Rendered:

  - Android (4–11)
  - iOS (10–14) [Closed pull requests with Bitcode-related changes](https://github.com/cossacklabs/themis/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Bitcode+%F0%9F%90%99%22)
  - Windows (experimenal MSYS2 support)

I don't quite like how the badge is not aligned with the text baseline... Well, at least in Firefox.

Maybe this looks better? (With `<sub>` around the badge.)

  - Android (4–11)
  - iOS (10–14) <sub>[Closed pull requests with Bitcode-related changes](https://github.com/cossacklabs/themis/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Bitcode+%F0%9F%90%99%22)</sub>
  - Windows (experimenal MSYS2 support)

But it's a hack and it still breaks interline spacing :(

**Review by ilammy — APPROVED** [review]
Well, I intended it as a joke, but alrighty 👍

File: README.md [hunk 2]
```diff
@@ -93,14 +93,14 @@ Themis is available for the following languages/platforms, refer to [language ho
...
   - Windows (experimenal MSYS2 support)
```
Comment (ilammy) [inline README.md:104]: Oh, and while we're at it...
```suggestion
  - Windows (experimental MSYS2 support)
```

File: README.md (hunk 1 above)
Comment (ilammy) [reply README.md:103]: Granted, it's a nice joke for a chat 😏 But I don't feel this looks nice enough here, or that our users share or empathize with the maintainers' pain of supporting Bitcode. If you'd still like to keep it, let's look for a better place for those vanity metrics? How about the **Languages** section, right below the Greater Table of Platforms?

Comment (vixentael) [reply README.md]: Listening to your advice, I've moved this badge to the very bottom of a page, giving it more like an easter egg flavour.

**Review by ilammy — APPROVED** [review]

**Pull request merged**